### PR TITLE
Slightly whiter background

### PR DESCRIPTION
**Pull request by OceanOak**
Before/ After
Frame 2

### CODE REVIEW

File: tailwind.config.js [hunk 1]
```diff
@@ -18,7 +18,7 @@ module.exports = {
     },
     extend: {
       colors: {
-        "light-background": "#F1F1F1",
+        "light-background": "#F8F8F8",
```
Comment (OceanOak) [inline tailwind.config.js:21]: Please let me know if we should make it whiter

Comment (pbiggar) [reply tailwind.config.js:21]: This looks good, but i think whiter might look better. Can you post a comparison?

Comment (OceanOak) [reply tailwind.config.js:21]: Frame 2 (1)

Comment (pbiggar) [reply tailwind.config.js:21]: Ok probably F8 is best!

Comment (OceanOak) [reply tailwind.config.js:21]: Here are some that are closer to F8.
Frame 3

Comment (pbiggar) [reply tailwind.config.js:21]: FA maybe?

Comment (OceanOak) [reply tailwind.config.js:21]: I think FA looks nice! (FWIW F8 is part of our original color palette). This updates the code snippets backgrounds to go with the new background color

Screenshot 2023-10-21 at 18 25 50

Screenshot 2023-10-21 at 18 03 12

**Pull request merged**